charts/kube-prometheus-stack/charts/grafana/values.yaml

```diff
@@ -630,12 +630,12 @@ sidecar:
     # If specified, the sidecar will search for dashboard config-maps inside this namespace.
     # Otherwise the namespace in which the sidecar is running will be used.
     # It's also possible to specify ALL to search in all namespaces
-    searchNamespace: null
+    searchNamespace: ALL
     # search in configmap, secret or both
     resource: both
     # If specified, the sidecar will look for annotation with this name to create folder and put graph here.
     # You can use this parameter together with `provider.foldersFromFilesStructure`to annotate configmaps and create folder structure.
-    folderAnnotation: null
+    folderAnnotation: metalk8s.scality.com/grafana-folder-name
```

Comment:
Is the `-name` suffix needed? I mean, do we expect some other annotations regarding Grafana folders?

Reply:
We don't, it just seems better to me to explain what impact this annotation has on the folder. I can change it if you'd like.